contracts/Configuration.sol

```diff
@@ -9,7 +9,6 @@ struct MarketplaceConfig {
 }
 
 struct CollateralConfig {
-  uint256 initialAmount; // amount of collateral necessary to fill a slot
   uint256 minimumAmount; // frees slot when collateral drops below this minimum
   uint256 slashCriterion; // amount of proofs missed that lead to slashing
   uint256 slashPercentage; // percentage of the collateral that is slashed
```

Comment:
Thinking about this a bit more, we probably want to move these other collateral related constants into the `Request` struct as well.

We don't want to enforce a network-wide minimum amount of collateral, so moving `minimumAmount` to `Request` makes sense. As for `slashCriterion` and `slashPercentage`, we could go either way with these. They could either be network-wide constants, or set per request.